container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -279,7 +281,8 @@ instance_groups:
           internal: 443
   configuration:
     templates:
-      properties.route_registrar.routes: '[{"name":"bits-service","tls_port":443,"uris":["bits.((DOMAIN))", "registry.((DOMAIN))"],"registration_interval":"20s","server_cert_domain_san":"bits","tags":{"component":"bits-service"}}]'
+      properties.bits-service.registry_endpoint: "https://127.0.0.1"
+      properties.route_registrar.routes: '[{"name":"bits-service","tls_port":443,"uris":["bits.((DOMAIN))"],"registration_interval":"20s","server_cert_domain_san":"bits","tags":{"component":"bits-service"}}]'
```

Comment:
We're removing `registry.((DOMAIN))` here right? Do we no longer need it?

Reply:
Yes, we don't want to expose the registry externally. At least now it has TLS support; when I started this commit it only supported HTTP.

The only reason to support the external endpoint would be as some mechanism to "export" apps as docker images from the cluster. I think that use case is misguided, as you cannot properly show what went into building this image.